common/djangoapps/third_party_auth/saml_configuration/serializers.py

```diff
@@ -0,0 +1,13 @@
+"""
+Serializer for SAMLConfiguration
+"""
+
+from rest_framework import serializers
+
+from third_party_auth.models import SAMLConfiguration
+
+
+class SAMLConfigurationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SAMLConfiguration
+        fields = ('id', 'slug')
```

Comment:
newline needed?

Reply:
It would show with a red circle with a line through it if I was missing a line here.